src/stories/3-fields/submit-button/submit-button.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import { ArgsTable, Description, Heading, PRIMARY_STORY, Stories, Title } from "@storybook/addon-docs";
+import { Meta, Story } from "@storybook/react/types-6-0";
+import React from "react";
+import { FrontendEngine, ISubmitButtonSchema, SubmitButton } from "../../..";
+import { ExcludeReactFormHookProps } from "../../common";
+
+const ExcludeDefaultProps = {
+	type: { table: { disable: true } },
+	id: { table: { disable: true } },
+	title: { table: { disable: true } },
+	onClick: { table: { disable: true } },
+};
+
+export default {
+	title: "Field/SubmitButton",
+	component: SubmitButton,
+	parameters: {
+		docs: {
+			page: () => (
+				<>
+					<Title>SubmitButton</Title>
+					<Description>The primary call to action component</Description>
+					<Heading>Props</Heading>
+					<Description>
+						This component also inherits the
+						[HTMLButtonElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLButtonElement)
+						attributes.
+					</Description>
+					<ArgsTable story={PRIMARY_STORY} />
+					<Stories includePrimary={true} title="Examples" />
+				</>
+			),
+		},
+	},
+	argTypes: {
+		...ExcludeReactFormHookProps,
+		...ExcludeDefaultProps,
+		schema: {
```

Comment:
- no need to explain the schema prop since devs are expected to integrate via JSON schema only
and each prop can do away with the `schema.` prefix
- missing `type` prop (it should have `SUBMIT` value`)
- `id` and `type` needs to be marked as required
- for each prop, can also define the control / type / defaultValue for easy tinkering via the control

can refer to textarea story too